Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -1828,6 +1866,12 @@ namespace Babylon
             static constexpr auto JS_CLASS_NAME = "XRSession";
             static constexpr auto JS_EVENT_NAME_END = "end";
             static constexpr auto JS_EVENT_NAME_INPUT_SOURCES_CHANGE = "inputsourceschange";
+            static constexpr auto JS_EVENT_NAME_INPUT_SQUEEZE_START = "squeezestart";
```

Comment:
>JS_EVENT_NAME_INPUT_SQUEEZE_START [](start = 34, length = 33)

do we ever fire these new events?

Reply:
good catch, they are to be removed, i forgot. Thanks!